app/components/landing-page/template.hbs

```diff
@@ -0,0 +1,47 @@
+{{#if fetchData.isRunning}}
+    {{loading-indicator-page}}
+{{else}}
+    {{#if showDashboard}}
+        <div class='reviews-dashboard content'>
```

Comment:
If we're doing all components, I think it'd be nice to make the landing page and dashboard separate components. Maybe the logic to switch between them would make sense on the controller? Not super important, but now seems like a convenient time to clean this up, if you don't mind...

Reply:
The request for providers is what messes up the other pages. So I can move some stuff around but I don't think the logic to switch between them can go in the controller since it's called when the app refreshes no matter what page the user is on (that was the bug on the settings page).